bitnami/external-dns/templates/deployment.yaml

```diff
@@ -496,8 +496,12 @@ spec:
           ports:
             - name: http
               containerPort: {{ .Values.service.port }}
+          {{- if .Values.readinessProbe }}
           readinessProbe: {{ toYaml .Values.readinessProbe | nindent 12 }}
+          {{- end }}
+          {{- if .Values.livenessProbe }}
```

Comment:
Could you do the same thing with `livenessProbe`?

Reply:
done, thanks @Mauraza